mmv1/products/vmwareengine/PrivateCloud.yaml

```diff
@@ -282,3 +296,12 @@ properties:
         name: 'fqdn'
         description: |-
           Fully qualified domain name of the appliance.
+
+  - !ruby/object:Api::Type::Enum
+    name: 'type'
```

Comment:
Should this be an immutable field, or can it be changed without recreating the resource?